src/protocols/PhysicalLayer/PhysicalLayer.h

```diff
@@ -4,6 +4,20 @@
 #include "../../TypeDef.h"
 #include "../../Module.h"
 
+// common IRQ flags
+#define RADIOLIB_IRQ_TX_DONE                                    0x00
+#define RADIOLIB_IRQ_RX_DONE                                    0x01
+#define RADIOLIB_IRQ_PREAMBLE_DETECTED                          0x02
+#define RADIOLIB_IRQ_SYNC_WORD_VALID                            0x03
+#define RADIOLIB_IRQ_SYNC_WORD_ERROR                            0x04
+#define RADIOLIB_IRQ_HEADER_VALID                               0x05
+#define RADIOLIB_IRQ_HEADER_ERR                                 0x06
+#define RADIOLIB_IRQ_CRC_ERR                                    0x07
+#define RADIOLIB_IRQ_CAD_DONE                                   0x08
+#define RADIOLIB_IRQ_CAD_DETECTED                               0x09
+#define RADIOLIB_IRQ_TIMEOUT                                    0x0A
+#define RADIOLIB_IRQ_LR_FHSS_HOP                                0x0B
+
```

Comment:
I would propose to remove LR_FHSS_HOP IRQ since it is very specific to LoRa modules. The other ones can be applied in some way even to FSK modules.

Reply:
In that spirit, what about SyncWordError? That one is only implemented by SX128x but given that it is in the same spirit as the other IRQs I thought I'd add it.

Reply:
Agreed, let's remove it - it's not really that useful outside very specific cases (where you most likely wouldn't be using PHY anyway).